hadoop-crypto/src/main/java/com/palantir/crypto2/hadoop/StandaloneEncryptedFileSystem.java

```diff
@@ -86,6 +90,13 @@ public void initialize(URI uri, Configuration conf) throws IOException {
         KeyPair keyPair = getKeyPair(conf);
         KeyStorageStrategy keyStore = new FileKeyStorageStrategy(delegate, keyPair);
 
+        if (Crypto.isNativeCodeLoaded()) {
```

Comment:
do we have CI-automation in this project that runs tests etc with-vs-without native libraries?